package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@saucelabs/cypress-junit-plugin",
+  "description": "Sauce Cypress JUnit Plugin",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "npx rimraf lib && tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "lint": "prettier --check '**/*.{js,ts,mjs,cjs}' && eslint .",
+    "fmt": "prettier --write '**/*.{js,ts,mjs,cjs}'",
+    "release": "tsc && release-it --github.release",
+    "release:ci": "tsc && npm run release -- --ci --no-git.requireCleanWorkingDir",
+    "release:patch": "npm run release -- patch",
+    "release:minor": "npm run release -- minor",
+    "release:major": "npm run release -- major"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/saucelabs/cypress-junit-plugin.git"
+  },
+  "bugs": {
+    "url": "https://github.com/saucelabs/cypress-junit-plugin/issues"
+  },
+  "author": "devx <devx@saucelabs.com>",
+  "files": [
+    "lib"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.13.2"
+  },
+  "dependencies": {
+    "debug": "^4.3.4",
+    "fast-xml-parser": "^4.3.4",
```

Comment:
I can only find usage of this as a dev dependency in your tests, but not in the main codebase. Should demote this to `devDependencies`.